utils/_features.py

```diff
@@ -2506,5 +2506,13 @@ def api_security_endpoint_discovery(test_object):
         """
         return _mark_test_object(test_object, feature_id=483, owner=_Owner.asm)
 
+    @staticmethod
+    def otel_logs_enabled(test_object):
+        """OTEL logs are enabled
+
+        https://feature-parity.us1.prod.dog/#/?feature=484
```

Comment:
This URL is currently pointing to the OpenTelemetry Metrics API feature. Did you mean to create a new feature and point to that?

Reply:
It should be a new feature. I'll update this